R/shiny-options.R

```diff
@@ -81,6 +81,9 @@ getShinyOption <- function(name, default = NULL) {
 #' \item{shiny.error (defaults to `NULL`)}{This can be a function which is called when an error
 #'   occurs. For example, `options(shiny.error=recover)` will result a
 #'   the debugger prompt when an error occurs.}
+#' \item{shiny.error.unhandled (defaults to `NULL`)}{A function that will be
+#'   called when an unhandled error that will stop the app session occurs. This
+#'   function should take the error condition object as its first argument.}
```

Comment:
I think we need to make it very clear that the function will not prevent the session from exiting -- that the purpose of this function is for logging or cleanup, and the session will still close afterward.

Reply:
Great point, handled in 4e7e15d04ebbfe77c99ec43c987392f46767078e